commercial/app/views/containerWrapper.scala.html

```diff
@@ -1,25 +1,32 @@
-@(classNames: Seq[String],
-  headerTitle: String,
-  headerClassNames: Option[Seq[String]] = None,
-  bodyClassNames: Option[Seq[String]] = None)(kicker: Html)(blurb: Html)(ctas: Html)(stamp: Html)(innards: Html)
+@( classNames: Seq[String]
+ , dataLinkName: Option[String]
+ , rowLayout: Option[String] = None
+ , optKicker: Option[Html] = None
+ , optBlurb: Option[Html] = None
+ , optActions: Option[Html] = None
+ , optStamp: Option[Html] = None
+ , optBadge: Option[Html] = None
+ )(title: Html)(innards: Html)
```

Comment:
I haven't seen comma first anywhere else in the codebase. Should we be consistent?